runtime/CMakeLists.txt

```diff
@@ -573,7 +573,7 @@ endif()
 if(J9VM_MODULE_THREAD)
 	add_subdirectory(thread)
 endif()
-if(J9VM_INTERP_NATIVE_SUPPORT AND NOT OMR_ARCH_RISCV)
+if(J9VM_INTERP_NATIVE_SUPPORT AND NOT (OMR_ARCH_AARCH64 AND OMR_OS_OSX))
```

Comment:
JIT has been enabled on AArch64 macOS for couple of months now.
Please remove `AND NOT (OMR_ARCH_AARCH64 AND OMR_OS_OSX)`.